docs/graphql/manual/api-reference/schema-metadata-api/remote-relationships.rst

```diff
@@ -0,0 +1,234 @@
+.. meta::
+   :description: Manage remote relationships with the Hasura schema/metadata API
+   :keywords: hasura, docs, schema/metadata API, API reference, remote joins, remote relationships
+
+Schema/Metadata API Reference: Remote Relationships
+===================================================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+Remote Joins will ensure independent and related types in Remote Schemas can be joined natively via Hasura.
```

Comment:
Is there a reason why we call it remote joins here and remote relationships in the rest of the page? We should probably be consistent with that.

Reply:
Also, what does `independent` mean here? Can it be independent and related at the same time? :D

Reply:
@marionschleifer Yes, I will change "remote joins" to "remote relationships".

The independence here is referring to independent remote schemas and the related types here is referring to the data types of the join(For example:an Int field can only be joined to another Int field). 

It seems a bit confusing though, I will try to make it more clear